src/core_landice/mode_forward/mpas_li_thermal.F

```diff
@@ -2741,7 +2966,7 @@ subroutine basal_melt_grounded_ice(&
       real(kind=RKIND) :: excessWater           ! thickness of excess meltwater (m)
 
       real(kind=RKIND), parameter :: &
-           maxWaterfrac = 0.01_RKIND       ! maximum allowed water fraction; excess drains to bed
+           maxWaterfrac = 0.10_RKIND       ! maximum allowed water fraction; excess drains to bed
```

Comment:
There is no very specific reason to this value. When I did the benchmark experiments, I wanted to know a bit more of the increase of the water fraction instead of prescribing it to 1% if it exceeds it. I agree that we should put it in the namelist.

Reply:
A new "config_max_water_fraction"option is added in the "thermal_solver" section in the Registry